test/mpi/dtpools/configure.ac

```diff
@@ -50,21 +53,6 @@ AC_ARG_VAR([DTP_MIN_OBJ_ID],
 AC_ARG_VAR([DTP_MAX_OBJ_ID],
```

Comment:
I forgot about `DTP_MIN/MAX_OBJ_ID`. We are no longer using these. We should probably remove them from the configure

Reply:
Fixed.